.github/workflows/pluto.yml

```diff
@@ -0,0 +1,37 @@
+name: Run Pluto
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Dependency setup
+        run: |
+          set -x
+          sudo apt-get -y update
+          sudo apt-get install -y curl wget gnupg2 jq
+          source /etc/os-release
+          sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+          wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_${VERSION_ID}/Release.key -O- | sudo apt-key add -
+          curl https://baltocdn.com/helm/signing.asc | gpg --dearmor | sudo tee /usr/share/keyrings/helm.gpg > /dev/null
+          sudo apt-get install apt-transport-https --yes
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+          sudo apt-get update -qq -y
+          sudo apt-get -qq --yes install podman helm
+          plutover=$(curl --location --silent 'https://api.github.com/repos/FairwindsOps/pluto/releases/latest' | jq -r '.tag_name')
+          curl -sLo - https://github.com/FairwindsOps/pluto/releases/download/${plutover}/pluto_$(echo ${plutover} | tr -d v)_linux_amd64.tar.gz | tar -xvzf - pluto
+          sudo install -m755 pluto /usr/local/bin/pluto
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Render helm template charts for pluto to scan the generated resource api usage
```

Comment:
I believe these will run in parallel. You need to use -- or provide dependencies between steps

Reply:
Nevermind. Had argo on my mind

Reply:
No worries, feel free to install act and give this a run if you want to see what it does/how it works. I use act to test out my github actions for my own repos, its nice cause then you can test without pushing a branch or worrying about jenkins.

Reply:
Just beware its... chatty, the end is what matters.